src/client/a2/main/grid/html/html-grid.component.pug

```diff
@@ -7,6 +7,7 @@
             [ngClass]="pulseClass(beat, pulse, pulses, pulseIndex)")
           .actual.note([ngClass]="noteClass(pulses)", *ngIf="player.value(key, pulseIndex)")
   .top-fx.overlay
+    pixi-effects
```

Comment:
Currently the back-fx doesn't span larger than where the 2 strips are. Essentially it occupies the same space as the top-fx but is behind the "rectangles" where the pulses are. Can't really see it.
You can try setting pixi-effects there and make it non-transparent to visualize the space.